conf/drupal/config/editor.editor.rules_of_court_section.yml

```diff
@@ -144,6 +144,36 @@ settings:
         -
           label: Checklist
```

Comment:
I think we can remove this "checklist" option - this is the same as the disc option stylewise